### DISCUSSION TITLE
How to auto-refresh content of a PostgreSQL table datasource in a table

### DISCUSSION THREAD

**Discussion by canob** [Budibase/budibase#8269] [Help] [answered]
Hi everyone,

I have a simple question: is there a way to "auto refresh" the search of a query that fill a table for example every 30 seconds, or every minute? Our use case is an app that work as a central console for different ticket systems, and we need that the table be updated as "real-time" as possible.

Thanks for your help!

**Comment by shogunpurple** [comment, marked as answer]
https://github.com/Budibase/budibase/issues/3992

**Comment by canob** [comment]
Sorry, I didn't saw that open issue. Ok, so a lot of people need that feature, but at my understanding revieweing #3992 , is not on the roadmap right now.

Thanks.
Regards.
